pom.xml

```diff
@@ -21,7 +21,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.ballerinalang</groupId>
     <artifactId>connector-mongodb-parent</artifactId>
-    <version>0.8.3-SNAPSHOT</version>
+    <version>0.8.4-SNAPSHOT</version>
```

Comment:
Why are we updating the SNAPSHOT version ?

Reply:
I have updated this version since we need to release with the new changes. Don't we need to do this?

Reply:
When we release the connector, minor version will change automatically by Jenkins. Current version is `0.8.2`, hence when we release this, the new version will be `0.8.3`. We don't need to manually update the version, except it is a major version change (like `0.9.0`)